KIPs/kip-dreaft_sdk_common_architecture.md

```diff
@@ -0,0 +1,191 @@
+---
+kip: <to be assigned>
+title: Klaytn SDK Common Architecture
+author: Junghyun Colin Kim <colin.kim@groundx.xyz>, Jimin Kim <jasmine.kim@groundx.xyz>, Seonyong Kim <kale.kim@groundx.xyz>
+discussions-to: <URL>
+status: Draft
+type: <Standards Track | Meta | Informational>
+category (*only required for Standard Track): <Core | Networking | Storage | Interface | KCT | SDK | Application>
+created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
+requires (*optional): <KIP number(s)>
+replaces (*optional): <KIP number(s)>
+---
+
+## Simple Summary
+A new software architecture for Klaytn development environment which is shared by all Klaytn SDKs.
+
+## Abstract
+The following standard allows Klaytn's SDK, Caver, to implement a common architecture regardless of language.
+
+## Motivation
+The terms used in the Klaytn SDK are different, and there is no structure to implement when the SDK is developed with other languages.
+
+The common architecture is designed to unify the terms used in the SDK and easily extend to other languages by establishing a language independent common architecture.
+
+## Specification
+
+### Terminology
```

Comment:
Hmm.. It looks like those descriptions are package descriptions, not descriptions of terms.

Reply:
Then, is it better to delete this part and include it when explaining the diagram for each package?